networkx/tests/test_convert_scipy.py

```diff
@@ -213,7 +212,7 @@ def test_from_scipy_sparse_matrix_parallel_edges(self):
         creating a multigraph.
 
         """
-        A = sp_sparse.csr_matrix([[1, 1], [1, 2]])
+        A = sp.sparse.csr_matrix([[1, 1], [1, 2]])
```

Comment:
```suggestion
        import scipy.sparse
        A = sp.sparse.csr_matrix([[1, 1], [1, 2]])
```
As an alternative to the above comment (if necessary).